test/generation.test.ts

```diff
@@ -19,15 +20,15 @@ describe('Generation tests', () => {
         httpVersion: '2',
     });
 
-    test('Generates headers', () => {
+    test('Generates headers basic', () => {
```

Comment:
Not sure what purpose does this change serve :D

Reply:
When you run it from ide, it runs all the tests that include a specific string. so `Generates headers` triggered like 3 tests 😄